tests/e2e/taqueria-plugin-taquito.spec.ts

```diff
@@ -0,0 +1,345 @@
+import {generateTestProject} from "./utils/utils";
+import fs from "fs";
+import {execSync} from "child_process";
+import path from "path";
+import waitForExpect from "wait-for-expect";
+import { TezosToolkit } from '@taquito/taquito';
+
+const tezos = new TezosToolkit('https://hangzhounet.api.tez.ie');
```

Comment:
Since there is only 1 describe block I would add these things below there to break up things getting imported and things getting initialized

Reply:
Sure. I can do it. Fixed.